profile_logins.go

```diff
@@ -0,0 +1,61 @@
+package linodego
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+)
+
+// Profile represents a Profile object
+type ProfileLogin struct {
+	Datetime   *time.Time `json:"_"`
```

Comment:
Is `Datetime` ignored intentionally?

Reply:
Initially I tried not ignoring it and was getting a parsing error: `2023/03/08 15:03:17.696151 ERROR RESTY parsing time "\"2018-01-02T03:04:05\"" as "\"2006-01-02T15:04:05Z07:00\"": cannot parse "\"" as "Z07:00"`

Reply:
I took care of this! I needed to add an `UnmarshalJSON` helper method to parse the `Datetime`.